editor/src/components/canvas/canvas-strategies/strategies/group-conversion-helpers.ts

```diff
@@ -687,6 +682,17 @@ export function convertGroupToFrameCommands(
   ]
 }
 
+export type GroupChildElement = JSXElementLike | JSXConditionalExpression
+
+export function elementCanBeAGroupChild(
+  element: JSXElementChild | null,
+): element is GroupChildElement {
+  if (element == null) {
+    return false
+  }
+  return isJSXElementLike(element) || isJSXConditionalExpression(element)
```

Comment:
Shouldn't we check if the active branch  of the conditional expression can be a group child?

Reply:
I think this would deserve some thinking: you can end up with a inactive branch that is not a valid child, and flipping the conditional would end up with an invalid group anyways, so it would end up in the same scenario 🤔

Reply:
makes sense. would this work? [`47cfc23` (#4109)](https://github.com/concrete-utopia/utopia/pull/4109/commits/47cfc23d9f578b0225d4101078fe656ff0d73d0e)